packages/models/src/lib/upload-config.unit.test.ts

```diff
@@ -1,48 +1,48 @@
 import { describe, expect, it } from 'vitest';
-import { UploadConfig, uploadConfigSchema } from './upload-config';
+import { uploadConfigSchema } from './upload-config';
 
 describe('uploadConfigSchema', () => {
   it('should accept a valid upload configuration', () => {
     expect(() =>
-      uploadConfigSchema.parse({
+      uploadConfigSchema().parse({
         apiKey: 'API-K3Y',
         organization: 'code-pushup',
         project: 'cli',
         server: 'https://cli-server.dev:3800/',
-      } satisfies UploadConfig),
```

Comment:
It is hard to properly type the configurable schema, therefor i just removed the type here

Reply:
reverted